app/views/templates/report-details.html.slim

```diff
@@ -104,7 +105,7 @@
               |  {{ result.duration | durationFormat }}
           pre ng-if='result.message' {{ result.message }}
     p.empty
-      span ng-if='noMoreResults' All results have been loaded. 
+      span ng-if='noMoreResults' All results have been loaded.
```

Comment:
Add a `'` between the `span` and the `em` to ensure there is at least one space in the case where the two messages are displayed at the same time.